.drone.yml

```diff
@@ -1164,6 +1164,60 @@ volumes:
   host:
     path: /var/run/docker.sock
 
+trigger:
+  event:
+  - promote
+---
+kind: pipeline
+name: promote-docker-image
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: docker-image-promote
+  image: quay.io/skopeo/stable:v1.1.1
+  commands:
+  - echo $${DOCKER_PASSWORD} | skopeo login docker.io --username $${DOCKER_USERNAME} --password-stdin
+  - skopeo copy docker://rancher/rancher:$${SOURCE_TAG} docker://rancher/rancher:$${DESTINATION_TAG} --all
```

Comment:
How do source_tag and destination_tag get defined?

Reply:
How is the script getting executed in drone? Does it still need to be run manually? I am not seeing the script reference in drone.yaml. Sorry I am curious about the workflow.

Reply:
It is manual yes, similar to the promoting of the chart.

Reply:
First step is to avoid manual pushing to DockerHub, similar to the chart promote, we use Drone CLI to promote a build which then can use the secrets in Drone to do the work.

For longer term, as this is an ad-hoc tasks, we should look at a workflow (engine) to be able to be triggered which will run what I included in this PR automatically based on input variables. (basically move the execution of the script to a web form that asks for 2 input variables)